ReactiveObjC/RACTuple.h

```diff
@@ -111,15 +115,84 @@ NS_ASSUME_NONNULL_BEGIN
 - (nullable id)objectAtIndexedSubscript:(NSUInteger)idx;
 @end
 
+/// A tuple with exactly two generic values.
+@interface RACTwoTuple<__covariant First, __covariant Second> : RACTuple
```

Comment:
Just as I pressed "Add comment" I came up with an argument for `RACOneTuple`, namely that it allows us to have a tuple with one value (as one sometimes wants) and specify the type of that value, which a plain `RACTuple`, of course, doesn't allow for.